app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
This feels odd to me and somewhat without precedent. Keyword arguments are intended to solve connascence of name. Sure, if it provides hints then great, but then aren't we saying the method isn't suitably named?